guide/getting-started/open-design.md

```diff
@@ -50,67 +50,66 @@ Illustration sources
 
 * help design the interface of your favorite open-source project
-* be interested in a specific topic and collaborate with multiple people and projects in related areas
-* create a design resource library (like an icon or illustration set, or a UI Kit) for others to use
-* help test and provide feedback to an open-source product
-* [contribute](/guide/contribute/) to this Bitcoin Design Guide (hint, hint)
+* collaborate with people and projects in areas related to specific topics of interest
+* create a design resource library like an icon or illustration set or a UI kit
+* help test an open-source product and provide feedback
+* [contribute](/guide/contribute/) to the Bitcoin Design Guide
 
-Whichever it may be, it is a voluntary, public contribution to efforts that you think are worthwhile. From these individual efforts, a network of people and projects emerges that is only loosely bound through personal relationships, communities, or the work itself. This is different from common corporate structures and requires individuals to think and act more independently and be comfortable with ambiguity.
+Whatever you choose to do, open design projects are voluntary public contributions. From this work, a network of people and projects emerge connected by personal relationships, specialized communities, and sharing the work. This is different from corporate structures that require individuals to think and act independently.
 
-Open-source development has existed and matured for decades, while open design is not widely known or practiced. Starting to design for Bitcoin products can sometimes be disorienting. Many designers need to adjust not just how they work and collaborate but also the relationship to their work. No longer is the design an artifact only controlled by just the designer or a small group of people inside an organization.
+In open-source software, public review and debate lead to better products and services. Open-source development has existed for decades, while open design is not widely known or practiced. Designing for bitcoin products, especially for newcomers in an open design setting, can be disorienting and require adjustments to how they work and collaborate. No longer is work the exclusive product of them or a small, localized team; instead, it becomes part of a public conversation.
 
-The design instead becomes part of a public conversation where anyone is invited to help shape it, and group consensus influences the way forward. In open-source software, this public review and debate often lead to better products and services, and this effect ideally carries over to open design.
-
-The best solutions typically come from a comprehensive understanding of the problem to be solved, whether technical, human, societal, economic, or otherwise. This makes a multidisciplinary approach essential, with users, designers, developers, translators, project managers, and others working together.
```

Comment:
Why was this paragraph removed? Isn't it good to point out how important multidisciplinary collaboration is?

Reply:
I assume it seemed a bit repetitive, but I see it can be important to mention different kinds of contributions. I've put it back and slightly re-written it, to make it more concise, how does it seem now with https://github.com/BitcoinDesign/Guide/pull/290/commits/e1e2978ab41c6df2eaef38bbd512dbcceda0697b ?

Reply:
Looks good. Thank you.